app/service/data_svc.py

```diff
@@ -172,7 +174,14 @@ def __init__(self, dao, utility_svc):
         return sources
 
     async def explode_planners(self, criteria=None):
-        return await self.dao.get('core_planner', criteria=criteria)
+        planners = await self.dao.get('core_planner', criteria=criteria)
+        # De-serialize json params into a Dict if params are present
+        for p in planners:
```

Comment:
We can safely add default params into our existing planners/YMLs and not worry about backwards compatibility. We haven't had any reports of external planners being built, so I'm ok assuming that here.

Reply:
missed your note about the comment at first -- it's gone now 161d36bd09cafacff2bbe328104227cc0229ff9b 

Agree about your note about default params and changing existing planners -- but, was there an associated change you were suggesting with it?